package/kedro_viz/server.py

```diff
@@ -129,9 +191,30 @@ def commands():
     help="Whether to open viz interface in the default browser or not. "
     "Defaults to True.",
 )
-def viz(host, port, browser):
+@click.option("--load-file", default=None, type=click.Path(exists=True, dir_okay=False))
+@click.option(
+    "--save-file", default=None, type=click.Path(dir_okay=False, writable=True)
+)
```

Comment:
@tolomea Are these flags new features that we should mention in the release note?